docs/dotnet/dotnet-kestrel-metrics.md

```diff
@@ -138,6 +138,50 @@ different processes could be listening on TCP port 12345 and UDP port 12345.
+| `max_request_header_count_exceeded` | The HTTP request headers exceeded the maximum count limit. Configured by `KestrelServerLimits.MaxRequestHeaderCount`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `max_request_headers_total_size_exceeded` | The HTTP request headers exceeded the maximum total size limit. Configured by `KestrelServerLimits.MaxRequestHeadersTotalSize`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `min_request_body_data_rate` | Reading the request body timed out due to data arriving too slowly. Configured by `KestrelServerLimits.MinRequestBodyDataRate`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `min_response_data_rate` | Writing the response timed out because the client did not read it at the specified minimum data rate. Configured by `KestrelServerLimits.MinResponseDataRate`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `missing_stream_end` | The connection received an HTTP/2 `HEADERS` frame for trailers without a stream end flag. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `output_queue_size_exceeded` | The connection exceeded the output queue size and was closed with `INTERNAL_ERROR`. This can be caused by an excessive number of HTTP/2 stream resets. For more information, see [Microsoft Security Advisory CVE-2023-44487:](https://github.com/dotnet/runtime/issues/93303). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `request_headers_timeout` | Request headers timed out while waiting for headers to be received after the request started. Configured by `KestrelServerLimits.RequestHeadersTimeout`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `response_content_length_mismatch` | The HTTP response body sent data that didn't match the response's `content-length` header. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `server_timeout` | The connection timed out with the `IConnectionTimeoutFeature`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `stream_creation_error` | The HTTP/3 connection received a stream that it wouldn't accept. For example, the client created duplicate control streams. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `stream_reset_limit_exceeded` | The connection received an excessive number of HTTP/2 stream resets and was closed with `ENHANCE_YOUR_CALM`. For more information, see [Microsoft Security Advisory CVE-2023-44487:](https://github.com/dotnet/runtime/issues/93303). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `stream_self_dependency` | The connection received an HTTP/2 frame that caused a frame to depend on itself. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
were stream dependencies ever implemented?

Reply:
No. But validation is still there.